### PR TITLE
Improve syntax highlighting for HTMl, CSS, and numerical literals

### PR DESCRIPTION
**Pull request by parlough**
Closes https://github.com/dart-lang/site-www/issues/3685

Before:
image

After:
image

@kwalrath

### CODE REVIEW

File: src/_assets/css/_code_shared.scss [hunk 1]
```diff
@@ -69,9 +69,14 @@ pre.console-output {
...
-  .l { @extend .lit; } /* Literal */
+  .cp { @extend .typ; } /* Comment.Preproc */
+  .l, .m, .mi, .mf, .mh, .mx, .mb, .mo { @extend .lit; } /* Literal.* */
   .o { @extend .opn; } /* Operator */
   .n { @extend .typ; } /* Name */
   .na { @extend .kwd; } /* Name.Attribute */
   .no { @extend .lit; } /* Name.Constant */
+  .nt { @extend .tag; } /* Name.Tag */
+  .nl { @extend .kwd; } /* Name.Label */
+  .nl { @extend .kwd; } /* Name.Label */
```
Comment (kwalrath) [inline src/_assets/css/_code_shared.scss:80]:
```suggestion
  .nl { @extend .kwd; } /* Name.Label */
```

**Review by kwalrath — COMMENTED** [review]
Looks good, and doesn't seem like it should hurt anything.

Should we put these in alphabetic order?

File: src/_assets/css/_code_shared.scss [hunk 2]
```diff
@@ -70,13 +70,12 @@ pre.console-output {
   .p { @extend .pun; } /* Punctuation */
   .c, .c1, .cm { @extend .com; } /* Comment */
   .cp { @extend .typ; } /* Comment.Preproc */
-  .l, .m, .mi, .mf, .mh, .mx, .mb, .mo { @extend .lit; } /* Literal.* */
```
Comment (kwalrath) [inline src/_assets/css/_code_shared.scss]: Did you mean to omit .mh?

Otherwise, lgtm!

Comment (parlough) [reply src/_assets/css/_code_shared.scss]: Oops, thanks!

**Review by kwalrath — APPROVED** [review]
lgtm!

**Pull request merged**